homework011/channel-buffering.go

```diff
@@ -5,8 +5,20 @@ import (
 )
 
 func Task2() {
-	fmt.Println("homework011_2")
+	var i int
+	ch := make(chan string)
```

Comment:
```suggestion
	ch := make(chan string)
```

`var i int`
は定義しなくてもいいと思います。